src/action/invoice.js

```diff
@@ -62,6 +64,7 @@ class InvoiceAction {
       const response = await this._grpc.sendCommand('addInvoice', {
         value: toSatoshis(invoice.amount, settings),
         memo: invoice.note,
+        private: false,
```

Comment:
@tanx I think it's the opposite. If it's `private: true` it will encode routing hints in the invoice with private channels in order to assist the payer in reaching you. 

During my simnet test I wasn't able to pay the the invoice from `LND3 --non-private--> LND2 --private--> LND1`. When changing it to `true` I was able to generate an invoice from the app and pay it.

Reply:
@erkarl good catch! I fixed it.